FF13Rando/Randomizers/BattleRando.cs

```diff
@@ -287,6 +337,7 @@ public override void Randomize()
                     });
                 });
 
+                Randomizers.SetUIProgress("Randomizing multiple character set battles", 2, 3);
                 List<string> multiCharasetBattles = battleData.Where(battle => battle.Value.Charasets.Count > 1).Select(battle => battle.Key).ToList();
                 multiCharasetBattles.Shuffle().ForEach(id =>
                 {
```

Comment:
It won't let me select it but I think at least one other issue is in this block, where I'm using the btsc id to look through the intersection rather than the charaset one - although I feel like that should just lead to less shuffling rather than incorrect ids necessarily, since there shouldn't be any values in the btsc list that have ids which are charaspec ids rather than battle ids?